ui/components/NotificationCenter/metadata.js

```diff
@@ -42,6 +55,166 @@ const TitleLink = ({ href, children, ...props }) => {
   );
 };
 
+const UnsuccessfulEntityWithError = ({ modelName, error }) => {
+  const entityTypesAndQuantities = {};
+  error.name.forEach((name, idx) => {
+    if (name === modelName) {
+      entityTypesAndQuantities[error.entityType[idx]] =
+        entityTypesAndQuantities[error.entityType[idx]] + 1 || 1;
+    }
+  });
+
+  const isEntityFile =
+    modelName.includes('.yaml') || modelName.includes('.yml') || modelName.includes('.json');
+  const message = `Import did not occur for ${Object.entries(entityTypesAndQuantities)
+    .map(([key, value]) => `${value} ${value > 1 ? 'entities' : 'entity'} of type ${key}`)
+    .join(', ')}.`;
+  if (isEntityFile) {
+    return (
+      <>
+        <Typography variant="body1">
+          Import process for file {modelName} encountered error
+        </Typography>
+        <ErrorMetadataFormatter metadata={error.error} event={{}} />
+      </>
+    );
+  }
+
+  return (
+    <>
+      <Typography variant="body1">{message}</Typography>
+      <ErrorMetadataFormatter metadata={error.error} event={{}} />
+    </>
+  );
+};
+
+const imageCache = {};
+
+const checkImageExists = async (url) => {
```

Comment:
@captain-Akshay Don't we have a logic already to check if the image link is valid otherwise fallback to default logo?